thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -217,73 +218,123 @@ public Response computeSeverity(@NotNull @QueryParam("collection") String collec
    * Regenerates anomalies for each window separately
    *
    * As the anomaly result regeneration is a heavy job, we move the function from Dashboard to worker
-   * @param id anomaly function id
+   * @param ids a string containing multiple anomaly function ids, separated by comma (e.g. f1,f2,f3)
```

Comment:
It does not feel right to take comma separated string as path parameter. Is there a particular reason why we want to feed f1, f2 at the same time instead of firing two calls for f1 and f2 separately?